src/canmatrix/cli/compare.py

```diff
@@ -51,23 +53,29 @@ def cli_compare(matrix1, matrix2, verbosity, silent, check_comments, check_attri
         matrixX can be any of *.dbc|*.dbf|*.kcd|*.arxml|*.xls(x)|*.sym
     """
 
-    import canmatrix.log
-    root_logger = canmatrix.log.setup_logger()
+    root_logger = canmatrix_log.setup_logger()
 
     if silent:
         # Only print ERROR messages (ignore import warnings)
         verbosity = -1
-    canmatrix.log.set_log_level(root_logger, verbosity)
+    elif not verbosity:
+        # Info Message
+        verbosity = 1
+    else:
+        # Debug Message
+        verbosity = 2
+
+    canmatrix_log.set_log_level(root_logger, verbosity)
```

Comment:
we had (theoretically) multiple verbosity levels, now we only have one? Is that what we want?

Reply:
Sorry, this will be revert too.